draft-ietf-rtcweb-jsep.xml

```diff
@@ -3406,6 +3406,12 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           </list></t>
         </list></t>
 
+         <t> During the processing, if an error is returned, the
+         session is left in an undefined state. The applications needs
+         to resolve this by either termination the session or doing a
+         rollback to a previously defined state as described in <xref
```

Comment:
I don't think rollback is guaranteed to recover the session at this point. I think we either need to say we automatically perform rollback, or that the session is left in a bad state.